tools/api-tests/src/api-test/manuscript.test.ts

```diff
@@ -2,79 +2,197 @@ import { test, TestContext } from "ava-ts";
 import dashboard from "../xpub-api/dashboard";
 import submission from "../xpub-api/submission";
 import { defaultConfig } from "../xpub-api";
-import { Manuscript, ManuscriptInput} from "../generated/graphql";
-import { stripTypeNameFromJson  } from '../utils';
+import { Manuscript, ManuscriptInput } from "../generated/graphql";
+import { stripTypeNameFromJson } from "../utils";
```

Comment:
Not an issue, more a consistency thing I've just spotted, we're using single quotes elsewhere in the app so should probably stick to those?

Reply:
Ah yes, this subfolder is using it's own prettier config, I'm going to add tickets to integrate it more with the rest of the codebase, such as including it in CI, syncing the prettier, eslint (well, tslint) etc. with the main project